components/maintenance/firmware_update_puma.mdx

```diff
@@ -139,6 +139,23 @@ Replace `motor_controller_firmware_husky_001.bin` with whatever firmware file yo
 
 :::
 
+:::tip
+
+You may need change the permissions of the `sflash` program.
+You can do this by right-clicking on `sflash` and selecting `properties`.
+Change the permissions to `Allow executing file as a program`, and including it in the `domain users` group.
```

Comment:
The domain user group would be from a Clearpath managed machine that is part of the work domain.